ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp

```diff
@@ -221,7 +222,8 @@ void string_to_spv_func(const std::string& _name, const std::string& in_fname, c
     std::string target_env = (name.find("_cm2") != std::string::npos) ? "--target-env=vulkan1.3" : "--target-env=vulkan1.2";
 
     // disable spirv-opt for coopmat shaders for https://github.com/ggerganov/llama.cpp/issues/10734
-    std::string opt_level = coopmat ? "" : "-O";
+    // XXX do not submit - temporarily disabled for bfloat
+    std::string opt_level = "";//coopmat ? "" : "-O";
```

Comment:
I don't intend to submit this line as-is. spirv-opt needs a bit of work, hopefully that will be resolved soon.